README.md

```diff
@@ -115,15 +115,30 @@ Python SDK to create, explore and modify PFB files.
       -o OUTPUT, --output OUTPUT
       --name_from NAME_FROM
       --name_to NAME_TO
+      
+### Rename enum
+
+    usage: pypfb rename enum [-h] -i INPUT -o OUTPUT --field_name FILED_NAME --val_from VALUE_FROM
+                                   --val_to VALUE_TO
+    
+    optional arguments:
+      -h, --help            show this help message and exit
+      -i INPUT, --input INPUT
+      -o OUTPUT, --output OUTPUT
+      -- field_name FIELD_NAME
+      --val_from VALUE_FROM
+      --val_to VALUE_TO
+      
 
 ## Examples
```

Comment:
The whole section with examples should refer to PFB tool as `pfb`, which is added on package installation. Users shouldn't refer to `pypfb/__main__.py`.

Reply:
Fixed in 395749242c00d4562e524dd8c22e3c2ea8887231